sdk/core/System.Net.ClientModel/README.md

```diff
@@ -0,0 +1,41 @@
+# System.Net.ClientModel shared client library for .NET
+
+`System.Net.ClientModel` provides shared primitives, abstractions, and helpers for .NET service client libraries.
+
+`System.Net.ClientModel` allows client libraries built from its components to expose common functionality in a consistent fashion, so that once you learn how to use these APIs in one client library, you'll know how to use them in other client libraries as well.
```

Comment:
This section is missing boilerplate links to **Source code | Package | API refdocs**.

Reply:
Why do you say that when they are here: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/README.md ?

Reply:
If you mean just the source code, package and api ref docs links those don't exist yet since the package has never been released.  How do I check those in before hand and avoid the broken link error in CI?

I believe they were left out of the original beta for the same reason https://www.nuget.org/packages/System.Net.ClientModel/1.0.0-beta.1#readme-body-tab